src/System.Management.Automation/security/wldpNativeMethods.cs

```diff
@@ -48,16 +48,23 @@ private SystemPolicy()
         /// <returns>An EnforcementMode that describes the system policy.</returns>
         public static SystemEnforcementMode GetSystemLockdownPolicy()
         {
-            if (s_allowDebugOverridePolicy || (s_systemLockdownPolicy == null))
+            if (s_systemLockdownPolicy == null)
             {
                 lock (s_systemLockdownPolicyLock)
                 {
-                    if (s_allowDebugOverridePolicy || (s_systemLockdownPolicy == null))
+                    if (s_systemLockdownPolicy == null)
                     {
-                        s_systemLockdownPolicy = GetLockdownPolicy(null, null);
+                        s_systemLockdownPolicy = GetLockdownPolicy(path: null, handle: null);
```

Comment:
First call here, we still make the win32 call.

Reply:
if there is no policy, `s_allowDebugOverridePolicy` is true and `s_systemLockdownPolicy` is none.

Reply:
continued in next comment.

Reply:
> First call here, we still make the win32 call.

Yes, I believe that's needed, right?